Insert sql statement probabilistic erro ERROR:  syntax error at or near "valuesorg"

## Bug Report

**For English only**, other languages will not accept.

Before report a bug, make sure you have:

- Searched open and closed [GitHub issues](https://github.com/apache/shardingsphere/issues).
- Read documentation: [ShardingSphere Doc](https://shardingsphere.apache.org/document/current/en/overview).

Please pay attention on issues you submitted, because we maybe need more details. 
If no response anymore and we cannot reproduce it on current information, we will **close it**.

Please answer these questions before submitting your issue. Thanks!

### Which version of ShardingSphere did you use?
we find java version: java8, full_version=1.8.0_282
ShardingSphere-5.1.2-SNAPSHOT
Commit ID: 952c1feacb26bc03ea4ea14d9f82b0d5777732c2
Commit Message: Update shasum arguments in Release documents (#18344)
Branch: master
Build time: 2022-06-13T19:07:26+0000

### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?
ShardingSphere-Proxy

### Expected behavior
insert successfully

### Actual behavior
insert failed


```
ERROR:  syntax error at or near "valuesorg"
LINE 1: insert into cur_test_13_02 values(2,22,'Geoge','Hainan'),(32...
```

```
[INFO ] 2022-06-20 17:04:59.577 [Connection-321-ThreadExecutor] ShardingSphere-SQL - Actual SQL: ds_0 ::: insert into cur_test_13_02 valuesorg.apache.shardingsphere.sharding.rewrite.token.pojo.ShardingInsertValuesToken@4ce0227e;
[ERROR] 2022-06-20 17:04:59.590 [Connection-321-ThreadExecutor] o.a.s.p.f.c.CommandExecutorTask - Exception occur:
org.opengauss.util.PSQLException: [xx.xx.xx.xx:46438/xx.xx.xx.xx:14000] ERROR: syntax error at or near "valuesorg"
  Position: 28
        at org.opengauss.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2901)
        at org.opengauss.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2630)
        at org.opengauss.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:362)
        at org.opengauss.jdbc.PgStatement.runQueryExecutor(PgStatement.java:562)
        at org.opengauss.jdbc.PgStatement.executeInternal(PgStatement.java:539)
        at org.opengauss.jdbc.PgStatement.execute(PgStatement.java:397)
        at org.opengauss.jdbc.PgStatement.executeWithFlags(PgStatement.java:339)
        at org.opengauss.jdbc.PgStatement.executeCachedSql(PgStatement.java:325)
        at org.opengauss.jdbc.PgStatement.executeWithFlags(PgStatement.java:302)
        at org.opengauss.jdbc.PgStatement.execute(PgStatement.java:298)
        at org.opengauss.jdbc.PgStatement.execute(PgStatement.java:1458)
        at com.zaxxer.hikari.pool.ProxyStatement.execute(ProxyStatement.java:103)
        at com.zaxxer.hikari.pool.HikariProxyStatement.execute(HikariProxyStatement.java)
        at org.apache.shardingsphere.proxy.backend.communication.jdbc.executor.callback.impl.ProxyStatementExecutorCallback.execute(ProxyStatementExecutorCallback.java:40)
        at org.apache.shardingsphere.proxy.backend.communication.jdbc.executor.callback.ProxyJDBCExecutorCallback.executeSQL(ProxyJDBCExecutorCallback.java:73)
        at org.apache.shardingsphere.proxy.backend.communication.jdbc.executor.callback.ProxyJDBCExecutorCallback.executeSQL(ProxyJDBCExecutorCallback.java:66)
        at org.apache.shardingsphere.proxy.backend.communication.jdbc.executor.callback.ProxyJDBCExecutorCallback.executeSQL(ProxyJDBCExecutorCallback.java:44)
        at org.apache.shardingsphere.infra.executor.sql.execute.engine.driver.jdbc.JDBCExecutorCallback.execute(JDBCExecutorCallback.java:86)
        at org.apache.shardingsphere.infra.executor.sql.execute.engine.driver.jdbc.JDBCExecutorCallback.execute(JDBCExecutorCallback.java:66)
        at org.apache.shardingsphere.infra.executor.kernel.ExecutorEngine.syncExecute(ExecutorEngine.java:135)
        at org.apache.shardingsphere.infra.executor.kernel.ExecutorEngine.parallelExecute(ExecutorEngine.java:131)
        at org.apache.shardingsphere.infra.executor.kernel.ExecutorEngine.execute(ExecutorEngine.java:116)
        at org.apache.shardingsphere.infra.executor.sql.execute.engine.driver.jdbc.JDBCExecutor.execute(JDBCExecutor.java:65)
        at org.apache.shardingsphere.proxy.backend.communication.jdbc.executor.ProxyJDBCExecutor.execute(ProxyJDBCExecutor.java:71)
        at org.apache.shardingsphere.proxy.backend.communication.ProxySQLExecutor.useDriverToExecute(ProxySQLExecutor.java:171)
        at org.apache.shardingsphere.proxy.backend.communication.ProxySQLExecutor.execute(ProxySQLExecutor.java:132)
        at org.apache.shardingsphere.proxy.backend.communication.ProxySQLExecutor.execute(ProxySQLExecutor.java:126)
        at org.apache.shardingsphere.proxy.backend.communication.jdbc.JDBCDatabaseCommunicationEngine.execute(JDBCDatabaseCommunicationEngine.java:140)
        at org.apache.shardingsphere.proxy.backend.communication.jdbc.JDBCDatabaseCommunicationEngine.execute(JDBCDatabaseCommunicationEngine.java:73)
        at org.apache.shardingsphere.proxy.backend.text.data.impl.SchemaAssignedDatabaseBackendHandler.execute(SchemaAssignedDatabaseBackendHandler.java:63)
        at org.apache.shardingsphere.proxy.frontend.opengauss.command.query.simple.OpenGaussComQueryExecutor.execute(OpenGaussComQueryExecutor.java:76)
        at org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask.executeCommand(CommandExecutorTask.java:107)
        at org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask.run(CommandExecutorTask.java:77)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)

```


### Reason analyze (If you can)

### Steps to reproduce the behavior, such as: SQL to execute, sharding rule configuration, when exception occur etc.
```
drop table if exists cur_test_13_01;
drop table if exists cur_test_13_02;
create table cur_test_13_01(c_id int,c_num int,c_name varchar(10),c_city varchar(10),c_add varchar(20));
insert into cur_test_13_01 values(1,18,'Allen','Beijing','AAAAABAAAAA'),(21,368,'Bob','Shanghai','AAAAACAAAAA'),
                           (3,59,'Cathy','Shenzhen','AAAAADAAAAA'),(4,96,'David','Suzhou','AAAAAEAAAAA'),
                           (5,17,'Edrwd','Fenghuang','AAAAAFAAAAA'),(61,253,'Fendi','Changsha','AAAAAGAAAAA');

create table cur_test_13_02(c_id int,c_num int,c_name varchar(10),c_city varchar(10));
--this sql probabilistic erro
insert into cur_test_13_02 values(2,22,'Geoge','Hainan'),(32,54,'Hebe','Taiwan'),
                           (12,59,'Ivail','HongKong'),(24,946,'Jack','Shanghai'),
                           (51,117,'Kathy','Qinghai'),(26,253,'Laura','Xian');

start transaction;
cursor cursor13 for
select c_name from cur_test_13_01 where c_id like '%1'
union
select c_name from cur_test_13_02 where c_id like '2%' order by 1;

fetch forward from cursor13;
close cursor13;
end;

drop table cur_test_13_01;
drop table cur_test_13_02;

```
### Example codes for reproduce this issue (such as a github link).

Steps:

```sql
INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES (720, 'Customer#000000720', '8shFEL7J9sq1NJXR8dixBqaTO,kbSx', 22, '32-575-838-4260', 5357.32, 'HOUSEHOLD ', 'furiously above the furiously ironic c');
```

1. Create a sharding rule without table `customer` like the following.

```yaml
schemaName: tpch_db

dataSources:
  ds_0:
    url: jdbc:opengauss://127.0.0.1:5433/tpch_0?loggerLevel=OFF
# ...
  ds_1:
    url: jdbc:opengauss://127.0.0.1:5433/tpch_1?loggerLevel=OFF
# ...

rules:
  - !SHARDING
    tables:
      lineitem:
        actualDataNodes: ds_${0..1}.lineitem
#      customer:
#        actualDataNodes: ds_${0..1}.customer

```

2. Execute SQL
```bash
gsql -h 127.0.0.1 -p 55433 -U gaussdb tpch_db -WPassword@123 \
-c "INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES (720, 'Customer#000000720', '8shFEL7J9sq1NJXR8dixBqaTO,kbSx', 22, '32-575-838-4260', 5357.32, 'HOUSEHOLD ', 'furiously above the furiously ironic c');"
```

3. Got error
```
[INFO ] 2022-09-24 11:39:43.922 [Connection-1-ThreadExecutor] ShardingSphere-SQL - Logic SQL: INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES (720, 'Customer#000000720', '8shFEL7J9sq1NJXR8dixBqaTO,kbSx', 22, '32-575-838-4260', 5357.32, 'HOUSEHOLD ', 'furiously above the furiously ironic c');
[INFO ] 2022-09-24 11:39:43.922 [Connection-1-ThreadExecutor] ShardingSphere-SQL - SQLStatement: OpenGaussInsertStatement(super=InsertStatement(super=AbstractSQLStatement(parameterCount=0, parameterMarkerSegments=[], commentSegments=[]), table=SimpleTableSegment(tableName=TableNameSegment(startIndex=12, stopIndex=19, identifier=IdentifierValue(value=customer, quoteCharacter=NONE)), owner=Optional.empty, alias=Optional.empty), insertColumns=Optional[InsertColumnsSegment(startIndex=21, stopIndex=108, columns=[ColumnSegment(startIndex=22, stopIndex=30, identifier=IdentifierValue(value=c_custkey, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=33, stopIndex=38, identifier=IdentifierValue(value=c_name, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=41, stopIndex=49, identifier=IdentifierValue(value=c_address, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=52, stopIndex=62, identifier=IdentifierValue(value=c_nationkey, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=65, stopIndex=71, identifier=IdentifierValue(value=c_phone, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=74, stopIndex=82, identifier=IdentifierValue(value=c_acctbal, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=85, stopIndex=96, identifier=IdentifierValue(value=c_mktsegment, quoteCharacter=NONE), owner=Optional.empty), ColumnSegment(startIndex=99, stopIndex=107, identifier=IdentifierValue(value=c_comment, quoteCharacter=NONE), owner=Optional.empty)])], insertSelect=Optional.empty, values=[InsertValuesSegment(startIndex=117, stopIndex=265, values=[LiteralExpressionSegment(startIndex=118, stopIndex=120, literals=720), LiteralExpressionSegment(startIndex=123, stopIndex=142, literals=Customer#000000720), LiteralExpressionSegment(startIndex=145, stopIndex=176, literals=8shFEL7J9sq1NJXR8dixBqaTO,kbSx), LiteralExpressionSegment(startIndex=179, stopIndex=180, literals=22), LiteralExpressionSegment(startIndex=183, stopIndex=199, literals=32-575-838-4260), LiteralExpressionSegment(startIndex=202, stopIndex=208, literals=5357.32), LiteralExpressionSegment(startIndex=211, stopIndex=222, literals=HOUSEHOLD ), LiteralExpressionSegment(startIndex=225, stopIndex=264, literals=furiously above the furiously ironic c)])]), withSegment=Optional.empty, onDuplicateKeyColumnsSegment=null)
[INFO ] 2022-09-24 11:39:43.923 [Connection-1-ThreadExecutor] ShardingSphere-SQL - Actual SQL: ds_0 ::: INSERT INTO customer (c_custkey, c_name, c_address, c_nationkey, c_phone, c_acctbal, c_mktsegment, c_comment) VALUES org.apache.shardingsphere.sharding.rewrite.token.pojo.ShardingInsertValuesToken@13d6d169;
[ERROR] 2022-09-24 11:39:43.961 [Connection-1-ThreadExecutor] o.a.s.p.f.c.CommandExecutorTask - Exception occur: 
org.opengauss.util.PSQLException: [127.0.0.1:33816/127.0.0.1:5433] ERROR: syntax error at or near "org"
  Position: 118
	at org.opengauss.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2901)
	at org.opengauss.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2630)
	at org.opengauss.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:362)
	at org.opengauss.jdbc.PgStatement.runQueryExecutor(PgStatement.java:562)
	at org.opengauss.jdbc.PgStatement.executeInternal(PgStatement.java:539)
	at org.opengauss.jdbc.PgStatement.execute(PgStatement.java:397)
	at org.opengauss.jdbc.PgStatement.executeWithFlags(PgStatement.java:339)
	at org.opengauss.jdbc.PgStatement.executeCachedSql(PgStatement.java:325)
	at org.opengauss.jdbc.PgStatement.executeWithFlags(PgStatement.java:302)
	at org.opengauss.jdbc.PgStatement.execute(PgStatement.java:298)
	at org.opengauss.jdbc.PgStatement.execute(PgStatement.java:1458)
	at com.zaxxer.hikari.pool.ProxyStatement.execute(ProxyStatement.java:103)
	at com.zaxxer.hikari.pool.HikariProxyStatement.execute(HikariProxyStatement.java)
	at org.apache.shardingsphere.proxy.backend.communication.jdbc.executor.callback.impl.ProxyStatementExecutorCallback.execute(ProxyStatementExecutorCallback.java:41)
	at org.apache.shardingsphere.proxy.backend.communication.jdbc.executor.callback.ProxyJDBCExecutorCallback.executeSQL(ProxyJDBCExecutorCallback.java:74)
	at org.apache.shardingsphere.proxy.backend.communication.jdbc.executor.callback.ProxyJDBCExecutorCallback.executeSQL(ProxyJDBCExecutorCallback.java:67)
	at org.apache.shardingsphere.proxy.backend.communication.jdbc.executor.callback.ProxyJDBCExecutorCallback.executeSQL(ProxyJDBCExecutorCallback.java:44)
	at org.apache.shardingsphere.infra.executor.sql.execute.engine.driver.jdbc.JDBCExecutorCallback.execute(JDBCExecutorCallback.java:95)
	at org.apache.shardingsphere.infra.executor.sql.execute.engine.driver.jdbc.JDBCExecutorCallback.execute(JDBCExecutorCallback.java:75)
	at org.apache.shardingsphere.infra.executor.kernel.ExecutorEngine.syncExecute(ExecutorEngine.java:135)
	at org.apache.shardingsphere.infra.executor.kernel.ExecutorEngine.parallelExecute(ExecutorEngine.java:131)
	at org.apache.shardingsphere.infra.executor.kernel.ExecutorEngine.execute(ExecutorEngine.java:116)
	at org.apache.shardingsphere.infra.executor.sql.execute.engine.driver.jdbc.JDBCExecutor.execute(JDBCExecutor.java:65)
	at org.apache.shardingsphere.proxy.backend.communication.jdbc.executor.ProxyJDBCExecutor.execute(ProxyJDBCExecutor.java:74)
	at org.apache.shardingsphere.proxy.backend.communication.ProxySQLExecutor.useDriverToExecute(ProxySQLExecutor.java:195)
	at org.apache.shardingsphere.proxy.backend.communication.ProxySQLExecutor.execute(ProxySQLExecutor.java:153)
	at org.apache.shardingsphere.proxy.backend.communication.ProxySQLExecutor.execute(ProxySQLExecutor.java:147)
	at org.apache.shardingsphere.proxy.backend.communication.jdbc.JDBCDatabaseCommunicationEngine.execute(JDBCDatabaseCommunicationEngine.java:130)
	at org.apache.shardingsphere.proxy.frontend.opengauss.command.query.simple.OpenGaussComQueryExecutor.execute(OpenGaussComQueryExecutor.java:76)
	at org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask.executeCommand(CommandExecutorTask.java:111)
	at org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask.run(CommandExecutorTask.java:78)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)

```

--- a/shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/main/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/ShardingInsertValuesToken.java
+++ b/shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/main/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/ShardingInsertValuesToken.java
@@ -40,6 +40,11 @@ public final class ShardingInsertValuesToken extends InsertValuesToken implement
         return result.toString();
     }
     
+    @Override
+    public String toString() {
+        return toString(null);
+    }
+    
     private void appendInsertValue(final RouteUnit routeUnit, final StringBuilder stringBuilder) {
         for (InsertValue each : getInsertValues()) {
             if (isAppend(routeUnit, (ShardingInsertValue) each)) {

--- a/shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/ShardingInsertValuesTokenTest.java
+++ b/shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/ShardingInsertValuesTokenTest.java
@@ -26,10 +26,10 @@ import org.apache.shardingsphere.sql.parser.sql.common.segment.dml.expr.simple.L
 import org.junit.Before;
 import org.junit.Test;
 
-import java.util.List;
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.LinkedList;
+import java.util.List;
 
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
@@ -53,13 +53,19 @@ public final class ShardingInsertValuesTokenTest {
         expressionSegment.add(expressionSegment1);
         expressionSegment.add(expressionSegment2);
         Collection<DataNode> dataNodes = new LinkedList<>();
+        dataNodes.add(new DataNode("logic_ds", "tbl_0"));
         ShardingInsertValue shardingInsertValue = new ShardingInsertValue(expressionSegment, dataNodes);
         List<InsertValue> insertValues = shardingInsertValuesToken.getInsertValues();
         insertValues.add(shardingInsertValue);
     }
     
     @Test
-    public void assertToString() {
+    public void assertToStringWithRouteUnit() {
         assertThat(shardingInsertValuesToken.toString(routeUnit), is("('shardingsphere', 'test')"));
+    }
+    
+    @Test
+    public void assertToStringWithoutRouteUnit() {
+        assertThat(shardingInsertValuesToken.toString(), is("('shardingsphere', 'test')"));
     }
 }

